config/environments/production.rb

```diff
@@ -32,7 +32,7 @@
   config.logstasher.suppress_app_log = true
 
   # Defaults to Rails.root.join("public/assets")
-  # config.assets.manifest = YOUR_PATH
+  config.assets.manifest = Rails.root.join("public/government/assets/assets-manifest.json")
```

Comment:
Is this correct in light of the comment above it.  That would suggest that this should be set to the directory containing the manifest, not the manifest itself...